core/worker.go

```diff
@@ -1178,6 +1178,31 @@ func (w *worker) prepareWork(genParams *generateParams, wo *types.WorkObject) (*
 		newWo.Header().SetInterlinkRootHash(interlinkRootHash)
 	}
 
+	if nodeCtx == common.PRIME_CTX {
+		if w.hc.IsGenesisHash(parent.Hash()) {
+			newWo.Header().SetQiToQuai(big.NewInt(0))
```

Comment:
is this also true in the case of expansion? just make sure this works there as well and not think in a 3x3 fixed topology setting